### ISSUE TITLE
Firebase - Service database is not available

### ISSUE THREAD

**Issue by matthewjumpsoffbuildings** [dmotz/trystero#56]
When trying to intialise a Firebase setup:
```
component.js:111 Uncaught Error: Service database is not available
    at Provider.getImmediate (component.js:111:15)
    at getDatabase (database.js:9780:45)
    at init (firebase.js:18:34)
    at firebase.js:21:14
    at crypto-36a3f9c0.js:30:10
```

I tried using the app ID, as well as the realtime database url, but to no avail.

According to this stack overflow it may be because the version of the Firebase SDK this project uses is out of date?

https://stackoverflow.com/questions/70823077/uncaught-error-service-database-is-not-available-firebase-javascript

**Comment by matthewjumpsoffbuildings** [comment]
I created this pull request https://github.com/dmotz/trystero/pull/57 that solves this issue, turns out the code to generate a database URL from the project ID was not working

**Comment by dmotz** [comment]
Fixed in [0.14.0](https://github.com/dmotz/trystero/releases/tag/0.14.0)
